### PR TITLE
Improve logging of tuned-adm recommend

### PR DESCRIPTION
**Pull request by TomasKorbar**
Hi,
recently we've noticed mutliple error messages warning user about running virt-what without root privileges. This pull request makes from these multiple messages one which says exactly what is wrong.
Problem occured when executing:
$ tuned-adm recommend

### CODE REVIEW

File: tuned/utils/profile_recommender.py [hunk 1]
```diff
@@ -32,7 +32,13 @@ def recommend(self, hardcoded = False):
 		profile = consts.DEFAULT_PROFILE
 		if hardcoded:
 			return profile
-		matching = self.process_config(consts.RECOMMEND_CONF_FILE)
+		has_root = os.getuid() == 0
```
Comment (olysonek) [inline tuned/utils/profile_recommender.py]: This should use `os.geteuid()`. The effective UID matters, not the real one.

File: tuned/utils/profile_recommender.py [hunk 2]
```diff
@@ -32,7 +32,13 @@ def recommend(self, hardcoded = False):
...
+		if not has_root:
+			log.warning(("Profile recommender is running without root"
+						 " privileges. Profiles with virt recommendation"
+						 " condition will be omitted."))
```
Comment (olysonek) [inline tuned/utils/profile_recommender.py]: I prefer to put log messages on a single line, even if it makes the line too long. It makes it easier to `grep` for the message.

File: tuned/utils/profile_recommender.py (hunk 1 above)
Comment (olysonek) [reply tuned/utils/profile_recommender.py]: @yarda Can you please fix that up once you merge this? I fixed one more new occurence of the problem here https://github.com/redhat-performance/tuned/pull/172

Comment (TomasKorbar) [reply tuned/utils/profile_recommender.py]: Fixed.

File: tuned/utils/profile_recommender.py (hunk 2 above)
Comment (TomasKorbar) [reply tuned/utils/profile_recommender.py]: Fixed

**Comment by olysonek** [comment]
Thanks, LGTM.

**Comment by yarda** [comment]
Thanks, merged.

**Pull request merged**